src/d/ir/expression.d

```diff
@@ -542,6 +542,10 @@ enum CastKind {
 	Invalid,
 	IntToPtr,
 	PtrToInt,
+	IntToFloat,
```

Comment:
They  should probably go after SPad, as visible in the different places where this is used.